packages/react-components/react-text/stories/Text/TextTruncate.stories.tsx

```diff
@@ -1,8 +1,21 @@
 import * as React from 'react';
-import { Text } from '@fluentui/react-components';
+import { makeStyles, shorthands, Text } from '@fluentui/react-components';
 
-export const Truncate = () => (
-  <Text truncate style={{ width: 100, overflow: 'hidden', whiteSpace: 'nowrap' }} block>
-    This is a really really really really long text
-  </Text>
-);
+const useStyles = makeStyles({
+  text: {
+    ...shorthands.overflow('hidden'),
+    width: '240px',
+    display: 'block',
+  },
+});
+
+export const Truncate = () => {
+  const styles = useStyles();
+
+  return (
+    <Text truncate wrap={false} className={styles.text}>
```

Comment:
what's the wrap=false for ?

Reply:
The `wrap={false}` will include the `whitespace: nowrap` and `overflow: hidden` to the styles.
![image](https://user-images.githubusercontent.com/1357885/200859459-d9bba567-4734-490e-bb8a-4e98a0798ae8.png)
![image](https://user-images.githubusercontent.com/1357885/200859599-4b5ee6f5-6bdb-42d3-83c7-3425a01de58f.png)

Reply:
gotcha ty